docs/get-started/roles-and-permissions/use_case.md

```diff
@@ -0,0 +1,41 @@
+# Use Case
```

Comment:
can we add a second use case with `service_name`? it should be simpler to understand.

Reply:
Ok.